api/nakadi-event-bus-api.yaml

```diff
@@ -1142,7 +1142,104 @@ paths:
         '204':
           description: Feature was successfully accepted.
 
+  /storages:
+    get:
+      tags:
+        - timelines-api
+      description: |
+        Lists all available storage backends.
+        The oauth resource owner username has to be equal to 'nakadi.oauth2.adminClientId' property
+        to be able to access this endpoint.
+      responses:
+        '200':
+          description: A list of all available storage backends.
+          schema:
+            type: array
+            description: list of storage backends.
+            items:
+              $ref: '#/definitions/Storage'
+        '403':
+          description: Access forbidden
+          schema:
+            $ref: '#/definitions/Problem'
+    post:
+      tags:
+        - timelines-api
+      description: |
+        Creates a new storage backend.
+        The oauth resource owner username has to be equal to 'nakadi.oauth2.adminClientId' property
+        to be able to access this endpoint.
+      parameters:
+        - name: storage
+          in: body
+          description: Storage description
+          schema:
+            $ref: '#/definitions/Storage'
+      responses:
+        '204':
```

Comment:
Why don't you want to use 201 here? Even in the description you use the word "created".